src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,63 @@
+package wooteco.subway.ui;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.domain.Line;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+@RestController
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(final LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        final Line createdLine = lineService.create(lineRequest);
+        LineResponse lineResponse = new LineResponse(createdLine);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId()))
+            .body(lineResponse);
+    }
+
+    @GetMapping("/lines")
+    public ResponseEntity<List<LineResponse>> findAllLine() {
+        final List<Line> lines = lineService.findAll();
+        final List<LineResponse> lineResponses = lines.stream()
+            .map(LineResponse::new)
+            .collect(Collectors.toList());
+        return ResponseEntity.ok(lineResponses);
+    }
```

Comment:
```suggestion
    public List<LineResponse> findAllLine() {
        final List<Line> lines = lineService.findAll();
        final List<LineResponse> lineResponses = lines.stream()
            .map(LineResponse::new)
            .collect(Collectors.toList());
        return lineResponses;
    }
```

상태코드를 200 OK로 반환한다면 이렇게 작성할 수 있습니다.

Reply:
list응답도 마찬가지로 작동되나보네요! 데이터 반환시 200 ok가 default라는 스프링부트의 기본기능을 활용하라고 페어도 추천해주셨었네요 감사합니다~!